BotDatabase.py

```diff
@@ -215,34 +290,45 @@ def RemoveBan(self, TargetId:int):
         except Exception as ex:
             Logger.Log(LogLevel.Error, f"Got error {str(ex)}")
             return BanLookup.DBError
-        
-        self.Database.execute(f"DELETE FROM banslist where Id={TargetId}")
+
+        stmt = select(Ban).where(Ban.target_discord_user_id==TargetId)
+        result = self.Database.execute(stmt).scalars(stmt).first()
+        if (result is None):
+            Logger.Log(LogLevel.Warn, f"Tried to remove nonexistant ban for {TargetId}")
+            return BanLookup.NotExist
+        else:
+            result.undelete()
+
         self.Database.commit()
-        
+
         return BanLookup.Good
     
     ### Getting Server Information ###
     def GetAllServersOfOwner(self, OwnerId:int):
-        ServersOwnedQuery = self.Database.execute(f"SELECT Activated, Id FROM servers WHERE OwnerId={OwnerId}")
-        return ServersOwnedQuery.fetchall()
+        stmt = select(Server).where(Server.discord_owner_user_id==OwnerId)
+        result = list(self.Database.execute(stmt).scalars(stmt).all())
+        if (not len(result)):
+            Logger.Log(LogLevel.Warn, f"Failed to load servers for given discord user id of: {OwnerId}!")
+        return result
     
     def GetOwnerOfServer(self, ServerId:int) -> int:
-        ServersOwnedQuery = self.Database.execute(f"SELECT OwnerId FROM servers WHERE Id={ServerId}")
-        return ServersOwnedQuery.fetchone()[0]
+        stmt = select(Server).where(Server.discord_server_id==ServerId)
+        result = self.Database.execute(stmt).scalars(stmt).first()
+        if (result is None):
+            Logger.Log(LogLevel.Warn, f"Tried to load owner for non existant server: {ServerId}!")
+        return result
     
     def GetAllBans(self, NumLastActions:int=0):
-        LimitStr:str = ""
-        if (NumLastActions > 0):
-            LimitStr = f" LIMIT {NumLastActions}"
-        BansListQuery = self.Database.execute(f"SELECT Id, BannerName FROM banslist ORDER BY ROWID DESC{LimitStr}")
-        return BansListQuery.fetchall()
+        stmt = select(Ban).order_by(desc(Ban.created_at))
+        if (NumLastActions):
+            stmt = stmt.limit(NumLastActions)
+        return self.Database.execute(stmt).scalars(stmt)
```

Comment:
Can this return a list wrap so we don't have to make the change in every file?

Reply:
Yes, will convert all returns to list()s